### DISCUSSION TITLE
Welcome to GEDCOM Discussions!

### DISCUSSION THREAD

**Discussion by clarkegj** [FamilySearch/GEDCOM#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dthaler** [comment]
Test: Does replying via email generate a post or not?

From: Gordon Clarke ***@***.***>
Sent: Tuesday, October 5, 2021 2:05 PM
To: FamilySearch/GEDCOM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [FamilySearch/GEDCOM] Welcome to GEDCOM Discussions! (Discussion #67)

👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  *   Ask questions you’re wondering about.
  *   Share ideas.
  *   Engage with other community members.
  *   Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.
